src/qt/walletmodel.cpp

```diff
@@ -549,10 +543,15 @@ bool WalletModel::bumpFee(uint256 hash, uint256& new_hash)
         DataStream ssTx{};
         ssTx << psbtx;
         GUIUtil::setClipboard(EncodeBase64(ssTx.str()).c_str());
-        Q_EMIT message(tr("PSBT copied"), tr("Copied to clipboard", "Fee-bump PSBT saved"), CClientUIInterface::MSG_INFORMATION);
+        Q_EMIT message(tr("PSBT copied"), tr("Fee-bump PSBT copied to clipboard"), CClientUIInterface::MSG_INFORMATION | CClientUIInterface::MODAL);
```

Comment:
At least on Ubuntu 22.04, now after the fee-bumping unsigned a modal pop-up is displayed:

![image](https://github.com/bitcoin-core/gui/assets/110166421/792ecd94-3662-4bfd-a048-095c5a6f790b)

while before there was a message sent thru the OS system information (?):

![image](https://github.com/bitcoin-core/gui/assets/110166421/950fbcab-b5f5-432f-9604-1cd55c8bbbeb)

I see in the code we usually use `CClientUIInterface::MSG_INFORMATION` (`transactionview.cpp`, `walletview.cpp`) and I'd prefer to keep this look & feel but at the same time I think having the modal window popping up it's making the user more aware of what just happened with the intention to accept it.

Reply:
The change was introduced because the OS system notification wasn't popping up on macOS. Either way is ok for me.